packages/fixie/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "fixie",
+  "version": "1.0.0",
+  "volta": {
+    "extends": "../../package.json"
+  },
+  "devDependencies": {
+    "@tsconfig/node18": "^2.0.1",
+    "@types/lodash": "^4.14.195",
+    "@types/node": "^20.3.1",
+    "@types/prompt-sync": "^4.2.0",
+    "@types/uuid": "^9.0.2",
+    "@types/yargs": "^17.0.24",
+    "@typescript-eslint/eslint-plugin": "^5.60.0",
+    "@typescript-eslint/parser": "^5.60.0",
+    "eslint": "^8.40.0",
+    "eslint-config-nth": "^2.0.1",
+    "typescript": "^5.1.3"
+  },
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "prod": "yarn build && node dist/serve-bin.js --port 8080",
+    "lint": "eslint . --max-warnings 0",
+    "lint:fix": "eslint . --fix",
+    "typecheck": "tsc -p tsconfig.json",
+    "build": "yarn run typecheck",
+    "bin:dev": "npx ts-node dist/serve-bin.js --port 8080 --package-path fixie" 
+  },
+  "dependencies": { 
+    "ai-jsx": "^0.7.3",
```

Comment:
```suggestion
"ai-jsx": "workspace:*",
```

Reply:
Since this is going to deploy to GCR independently of ai-jsx, I think this needs to have a specific revision that it can pull? Or am I overlooking something?

Reply:
maybe we postproc this file when building the container to replace "workspace" with the local version?

Reply:
If you do `npm pack` and it has a workspace reference will that automatically "link in" (i.e. copy) the files from the workspace into the tarball? Might be a question for @NickHeiner

Reply:
The answer seems to be "no". IMO sticking with a version that resolves through NPM is sensible.

Reply:
thanks for looking into that. going to just hack this to insert "latest" for the time being